### PR TITLE
{lib}[GCCcore/10.3.0] SSW v1.2.4

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/5cdda04629f2f537ded7de1f8d1d41a9 for a full test report.

File: easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+files_to_copy = [
+    (['ssw_test'], 'bin'),
+    (['*.%s*' % SHLIB_EXT], 'lib'),
+    (['*.h'], 'include/ssw'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ssw_test', 'lib/libssw.%s' % SHLIB_EXT],
+    'dirs': [],
+}
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb:46]: @SIB-software would you consider adding `sanity_check_commands`?

Comment (smoretti) [reply easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb:46]: There is no `sanity_check_commands` in the previous SSW-1.1.1 EB files.

I don't really know this tool, I just need it as requirement.
The simplest `sanity_check_commands` I see could be to display the help message but it does not exit with a 0 status code. So `ssw_test || true` would not be a very useful check.

@verdurin What do you think?

Comment (smoretti) [reply easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb:46]: Or something like `ssw_test 2>&1 | grep 'Usage:'`  ?

Comment (verdurin) [reply easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb:46]: Yes, that's a commonly-used approach.
I'm just trying to add these as easyconfigs are updated, where possible, as they're a nice extra check.
You are free to say it's too much work.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/2c7001c40efbea47572f015935ee7283 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @SIB-software!

**Pull request merged**